Makefile

```diff
@@ -853,6 +853,11 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
+mdlint:
+	@echo "Running Markdown linter..."
+	$(NODE) tools/remark-cli/cli.js -q \
+	  doc/api doc/api_assets doc/guides src lib benchmark tools/doc/ tools/icu/
```

Comment:
Yeah, that was about typos, which is pretty much the actual content — it makes sense not to change that.

And the fact that commit messages (initially in text) are converted to changelogs (which are Markdown) without proper escaping is mostly a tooling issue here (which is likely not worth fixing, though — those are very rare and the linter would catch that).